### PR TITLE
Change mentions of cuboid to N-dimension

### PR DESCRIPTION
**Pull request by grovesNL**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -878,7 +878,8 @@ struct Element<T> {
     next: MaybeIndex,
 }
 
-/// Index of coordinates in a cuboidal `grid_size.pow(DIM)` grid
+/// Index of coordinates in a `DIM`-dimensional grid. Each dimension is subdivided by `grid_size`,
+/// so the grid contains `grid_size.pow(DIM)` cells in total.
```
Comment (Ralith) [inline src/lib.rs]: The intent here is to describe a specific geometry, which "grid" doesn't accomplish. Perhaps "Index of coordinates in a grid of `DIM`-cubes" or similar?

**Review by Ralith — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (grovesNL) [reply src/lib.rs]: `DIM`-cubes sounds good.

File: src/lib.rs [hunk 2]
```diff
@@ -1112,7 +1113,7 @@ mod tests {
         );
     }
 
-    /// Iterator over all relative coordinates within a cuboid grid
+    /// Iterator over all relative coordinates within a `DIM`-dimensional grid
```
Comment (Ralith) [inline src/lib.rs]: As above?

**Review by Ralith — APPROVED** [review]
Thanks!

**Pull request merged**